### ISSUE TITLE
Is a "custom" tax table usable?

### ISSUE THREAD

**Issue by Golytelee** [joey711/phyloseq#1222]
Hi all,
I'm using phyloseq to analyze some metagenomic data for potential microbiome function analyses. I have a phyloseq object that has the KEGG orthology ID and per sample counts of annotated genes as the OTU table and the sample metadata. I'd like to have the gene names as "taxa" and have a table containing the KO and associated gene name. Is there a way to coerce this table into something phyloseq would recognize as a "tax table"?
Thanks in advance!

**Comment by mikemc** [comment]
Yes, you can create a tax table from any character matrix as long as the row names are the same as whatever you want to be the "taxa", as described in `help(tax_table)`. The columns don't have to correspond to taxonomic ranks; they can include any information about the "taxa" that you want.

**Comment by Golytelee** [comment]
Thanks mikemc, that worked perfectly, I was trying to just straight up import my matrix in the phyloseq construction command, I didn't realize I had to invoke tax_table() 1st then use that object. Dada2 handles that part as part of its workflow so I didn't realize there is another step.
Thanks again for your help!
